spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
Let's tidy up how we write our test data with the use of factories. That makes it easier for you guys to reuse those factories instead of having to write the code to create `user` object for instance (below) everytime you need it.

With factories you could have simply done the ff:

`let(:user) { create(:user) }`